src/webui/www/private/download.html

```diff
@@ -72,19 +72,23 @@ <h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT
                     </tr>
                     <tr>
                         <td>
-                            <label for="startTorrent">QBT_TR(Start torrent)QBT_TR[CONTEXT=AddNewTorrentDialog]</label>
+                            <label for="skip_checking">QBT_TR(Skip hash check)QBT_TR[CONTEXT=AddNewTorrentDialog]</label>
                         </td>
                         <td>
-                            <input type="hidden" id="startTorrentHidden" name="paused" />
-                            <input type="checkbox" id="startTorrent" />
+                            <input type="checkbox" id="skip_checking" name="skip_checking" value="true" />
                         </td>
                     </tr>
                     <tr>
                         <td>
-                            <label for="skip_checking">QBT_TR(Skip hash check)QBT_TR[CONTEXT=AddNewTorrentDialog]</label>
+                            <label for="addTorrentOption">QBT_TR(When torrent is added:)QBT_TR[CONTEXT=AddNewTorrentDialog]</label>
```

Comment:
I think this option should stay in its original place (swap back with "Skip hash check").
Applies to upload.html too.